src/schema/v2/Alerts/index.ts

```diff
@@ -484,3 +484,32 @@ export const AlertsSummaryFields = {
     resolve: (resp) => resp,
   },
 }
+
+export const PartnerAlertsFields = {
+  id: { type: GraphQLString },
```

Comment:
Be careful with `id` as that needs to conform to the GraphQL spec, this can likely just be left out.

Reply:
Replaced with the `IDFields` helper 🙏🏻